rogue-thi-app/lib/backend-utils/rooms-utils.js

```diff
@@ -244,6 +249,35 @@ export async function getRoomAvailability (roomRequestList, day = new Date()) {
   return processedOpenings
 }
 
+export async function getRoomCapacity (roomRequestList, day = new Date()) {
```

Comment:
`roomRequestList` is unused 

```suggestion
export async function getRoomCapacity (day = new Date()) {
```